bin/src/named.rs

```diff
@@ -469,7 +483,7 @@ fn main() {
     let tls_cert_config = config.get_tls_cert();
 
     // and TLS as necessary
-    // TODO: we should add some more control from configs to enable/disable TLS/HTTPS
+    // TODO: we should add some more control from configs to enable/disable TLS/HTTPS/Quic
```

Comment:
Nit: I think QUIC is always capitalized, why did you spell it as `Quic`?

Reply:
Oh, thanks, that was something I wanted to double check, I ran through this quickly...